components/molecule/rating/src/Icons/starFilled.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const StarFilled = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <path d="M0 0h24v24H0z" fill="none" />
```

Comment:
This path is not required. Also neither all the paths with fill="none"

Reply:
Thanks! I didn't know this 👍